### ISSUE TITLE
Error on first run

### ISSUE THREAD

**Issue by peduarte** [henriquea/movtogif-cli#1]
```
➜  Desktop movtogif cb-fusion-text.mov cb-fusion-text.gif
Starting the magic 🎩
Cooking 🍰
{ [Error: Command failed: convert -resize 75% -delay 8 -dither none -coalesce -layers optimize -depth 8 -colors 128 -loop 0 /var/folders/vv/djfnn7kn17g2rvm45nzmhmr80000gr/T/movtogif/gif*.png cb-fusion-text.gif
convert: unable to open image `/var/folders/vv/djfnn7kn17g2rvm45nzmhmr80000gr/T/movtogif/gif*.png': No such file or directory @ error/blob.c/OpenBlob/2705.
convert: unable to open file `/var/folders/vv/djfnn7kn17g2rvm45nzmhmr80000gr/T/movtogif/gif*.png' @ error/png.c/ReadPNGImage/3921.
convert: no images defined `cb-fusion-text.gif' @ error/convert.c/ConvertImageCommand/3257.
]
  killed: false,
  code: 1,
  signal: null,
  cmd: 'convert -resize 75% -delay 8 -dither none -coalesce -layers optimize -depth 8 -colors 128 -loop 0 /var/folders/vv/djfnn7kn17g2rvm45nzmhmr80000gr/T/movtogif/gif*.png cb-fusion-text.gif' }
```

**Comment by henriquea** [comment]
Fixed on [v1.0.4](https://github.com/henriquea/movtogif-cli/releases/tag/v1.0.4)

**Comment by peduarte** [comment]
Po mano
